### DISCUSSION TITLE
Initialization of JointJS+ in Angular

### DISCUSSION THREAD

**Discussion by MeeraVarsani01** [clientIO/joint#2712] [Q&A] [answered]
### Introduction

Can anybody tell me when will the JointJS+ library's functionalities loaded or initialize in an angular project. My trail license is expired so i get a popup at the initialization of an angular project , but i am using the library in a child component which loads with lazy loading. Is library loaded at initialization of an angular project or when i load the graph in my component?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment, marked as answer]
If you get a message at startup, it's not a lazy load.

JointJS is a Javascript library. It's more of a general question about how to lazy load an external JS library in Angular.  I recommend taking a look at the Angular forums.
